package.json

```diff
@@ -27,10 +27,10 @@
     "prettier-check": "prettier --check ."
   },
   "dependencies": {
-    "react-markdown": "^8.0.5",
-    "rehype-raw": "^6.1.1",
-    "rehype-sanitize": "^5.0.1",
-    "remark-gfm": "^3.0.1",
+    "react-markdown": "^6.0.3",
+    "rehype-raw": "^5.0.0",
+    "rehype-sanitize": "^4.0.0",
+    "remark-gfm": "^1.0.0",
```

Comment:
It looks like the older version of `remark-gfm` doesn't support footnotes. Do we need to add `remark-footnotes` if we want to support all current GFM syntax?

Reply:
I don't think we need footnote support. I checked the KG implementation and they don't have it.